lalrpop/src/lr1/codegen/parse_table.rs

```diff
@@ -332,6 +334,19 @@ impl<'ascent, 'grammar, W: Write> CodeGenerator<'ascent, 'grammar, W, TableDrive
                                                                       .zip(0..)
                                                                       .collect();
 
+        let state_type = {
+            // `reduce_indices` are allowed to be +1 since the negative maximum of any integer type
+            // is one larger than the positive maximum
+            let max_value = states.len().max(reduce_indices.len() - 1);
```

Comment:
you are using unstable library feature 'ord_max_min'

Reply:
Oops, didn't have time to run through the full tests, was sure that I had used this on stable already.